vuln/npm/399.json

```diff
@@ -0,0 +1,21 @@
+{
+  "id": "399",
+  "title": "Command Injection - Generic",
+  "overview": "`whereis` concatenates unsanitized input into exec() command",
+  "created_at": "2018-02-25",
```

Comment:
We use full datetime in other vulns. Not a big deal but we need to reach consistency at some point. I think that date as it is in this PR makes more sense.

Reply:
@dgonzalez Some come with full date, some not. This isn't the first vuln with just the date, without time information.

Reply:
yeah, let's keep to YYYY-MM-DD format for now until we're able to agree on a final structure for the file and how to manage it. Then we can work out migration paths.

Reply:
Super!